tsl/src/remote/async.c

```diff
@@ -162,6 +164,16 @@ async_request_send_internal(AsyncRequest *req, int elevel)
 	remote_connection_configure_if_changed(req->conn);
```

Comment:
@erimatnor Nit: This line is not touched by your PR. However, in Line 233, we check the return value of `remote_connection_configure_if_changed`. In this line, we are ignoring it. I would suggest checking the return value of the function also here.

Reply:
Fixed